server/utils/tenant-pages.ts

```
@@ -98,25 +91,12 @@ interface TenantPageVariantRow {
-  ever_published?: boolean | number | null
-  status?: string | null
-  published_revision_id?: string | null
-  document_published_revision_id?: string | null
-}): boolean {
-  return Boolean(
-    input.ever_published
-    || input.status === 'published'
-    || input.published_revision_id
-    || input.document_published_revision_id,
-  )
+export function tenantPageHasPublicationHistory(input: any): boolean {
```

ESLint

[error] 'input' is defined but never used. Allowed unused args must match /^_/u.

(no-unused-vars)